index.html

```diff
@@ -132,22 +131,22 @@ <h3>Multi-Post Storiese</h3>
               ever since the 1500s, when an unknown printer took a standard
               dummy text.
             </p>
-            <div id="tb3">
-              <span id="a">HTML</span>
-              <span id="b">CSS</span>
-              <span id="c">BOOTSTRAP</span>
-              <span id="d">RUBY</span>
+            <div id="tb2">
+              <span id="a2">HTML</span>
+              <span id="b2">CSS</span>
+              <span id="c2">BOOTSTRAP</span>
+              <span id="d2">RUBY</span>
             </div>
```

Comment:
- [ ] `required`
- [ ] Here as mentioned above 
- [ ] However the list of project technologies are supposed to `ul` and each of them in a `li` see example `below`
![image](https://user-images.githubusercontent.com/66230306/186690799-18f92099-0f01-46bc-8bf3-46ea9828594f.png)
Kindly make amends on all the technologies